src/v/config/config_store.h

```diff
@@ -70,42 +70,48 @@ class config_store {
 
         for (auto const& node : root_node) {
             auto name = node.first.as<ss::sstring>();
-            auto found = _properties.find(name);
-            if (found == _properties.end()) {
-                found = _aliases.find(name);
-                if (found == _aliases.end()) {
-                    if (!ignore_missing.contains(name)) {
-                        throw std::invalid_argument(
-                          fmt::format("Unknown property {}", name));
+            auto* prop = [&]() -> base_property* {
+                auto found = _properties.find(name);
+                if (found == _properties.end()) {
+                    found = _aliases.find(name);
+                    if (found == _aliases.end()) {
+                        return nullptr;
                     }
```

Comment:
nit: in the theme of reduced scope / readability, maybe easier to follow as

```
auto found = _properties.find(name);
if (found != properties.end()) {
  return found->second;
}
found = _aliases.find(name);
if (found != _aliases.end()) {
  return found->second;
}
return nullptr;
```

Reply:
true, i'll change it locally if i find a better way to test the chages